website/cue/reference/components/sinks.cue

```diff
@@ -6,6 +6,25 @@ components: sinks: [Name=string]: {
 	features: _
 
 	configuration: {
+		if features.acknowledgements != _|_ {
+			if features.acknowledgements.enabled {
+				acknowledgements: {
+					common:      true
+					description: "Controls how acknowledgements are handled by this sink. When enabled, all connected sources that support end-to-end acknowledgements will wait for the destination of this sink to acknowledge receipt of events before providing acknowledgement to the sending source. These settings override the global `acknowledgement` settings."
+					required:    false
+					type: object: options: {
+						enabled: {
+							common:      true
+							description: "Controls if all connected sources will wait for this sink to deliver the events before acknowledging receipt."
+							warnings: ["Disabling this option may lead to loss of data, as destination sinks may reject events after the source acknowledges their successful receipt."]
```

Comment:
I might leave this warning out until we make it the default. Or change the wording a bit to be something like "We recommend enabling this...".